src/components/layout/index.js

```diff
@@ -0,0 +1,60 @@
+// https://github.com/alphagov/govuk-frontend/blob/master/src/components/checkbox/_checkbox.scss
```

Comment:
this first line should be a link to the scss file (or other related component resources) that you are basing this component on